config/application.yml

```diff
@@ -6,6 +6,8 @@ case ENV['RAILS_ENV']
 
 when 'local'
 
+  ENV['MLN_SETS_URL'] = "dev-sets.mylibrarynyc.local"
+  ENV['MLN_URL'] = "dev-www.mylibrarynyc.local"
```

Comment:
Sorry, but could you change `MLN_URL` to `MLN_INFO_SITE_URL` please? It's a bit clearer. Thanks!